content/en/docs/faq/forgot-password.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Forgot Password"
```

Comment:
The **Forgot Password** should not be created as a single guide. I recommend you to change the title to **FAQ-account and access management**, it will be easier to maintain and add more related questions in the long term.

Reply:
Merge this guide first and I will adjust the location of this guide later.